content/validation/before/domains.md

```diff
@@ -0,0 +1,39 @@
+Data plays a crucial role in the process of training any ML model. It is something from which the model learns to solve a given task and therefore care needs to be taken with its handling:
+
+1. By making sure that a data set is relevant to the problem and no deviation from expectations is introduced: so that a resulted model doesn't incorporate these undesired discrepancies.
+2. By performing a proper preprocessing of the data set: so that the training step goes smoothly.
+
+In this section a general [domain perspective](https://en.wikipedia.org/wiki/Domain_adaptation) on data will be covered and in the following sections a more granular look will be taken from the side of features and construction of inputs to the model.
+
+## Coverage  
+To begin with, one needs to bear in mind that training data should be as close as possible to data they expect to have in the context of analysis. Speaking in more formal terms,
+> Domains of training (used to train the model) and inference (used to make final predictions) data sets should not sizeably diverge.
+
+??? example "Examples"
```

Comment:
Why did you put "??? example"? Do you need help with something or is this just a mistake?